Assets/Scripts/Utils.cs

```diff
@@ -13,7 +13,19 @@ public enum PlayerType {
         HUMAN,
         BOT
     }
+    public enum PlayerPlatform {
+        EDITOR,
+        OCULUS,
+        STEAMVR
+    }
+    public enum HandSide {
+        UNSPECIFIED,
+        LEFT,
+        RIGHT
+    }
 
-    public static Color blue = new Color(0, 0.5f, 1);
-    public static Color orange = new Color(0.937f, 0.447f, 0.0823f);
+    public static class Constants {
```

Comment:
In future, it might be possible to place some UI constants here too like fontsize and stuff